source/Random.h

```diff
@@ -41,6 +41,9 @@ class Random {
 	static uint32_t Binomial(uint32_t t, double p = .5);
 	// Get a normally distributed number (mean = 0, sigma= 1).
 	static double Normal();
+	// Return a number from [0,1] derived from a normal curve,
+	// compressed according to smoothness.
+	static double CompressedNormal(double smoothness);
```

Comment:
I think I had it that way?  You asked me to move it into Random.

Reply:
What I'm suggesting is that you use the [parameterized constructor](https://en.cppreference.com/w/cpp/numeric/random/normal_distribution/normal_distribution) instead of reinventing the wheel.

Reply:
That is an option that I believe I was using at one point, though it required me to generate the distribution in my logic as opposed to just getting a standard one.  It would change things a little beyond that, but it isn't going to eliminate the "fractional retention" (as I've been calling it for lack of a better term) as that is what facilitates the smoothness, through the overlap.  Not sure at what point you think I'm reinventing the wheel, though.